tests/unit/test_lambda_event_invoke_config.py

```diff
@@ -0,0 +1,67 @@
+import os
+import unittest
+
+from localstack.services.awslambda import lambda_api
+from localstack.utils.aws.aws_models import LambdaFunction
+from localstack.constants import LAMBDA_TEST_ROLE
+
+os.environ['AWS_ACCESS_KEY_ID'] = 'test'
+os.environ['AWS_SECRET_ACCESS_KEY'] = 'test'
+
+
+class TestLambdaEventInvokeConfig(unittest.TestCase):
```

Comment:
This test class better belongs in the existing unit test file `tests/unit/test_lambda.py` - can we please move it there?